tooling/bundler/src/bundle/windows/msi/wix.rs

```diff
@@ -788,8 +785,9 @@ pub fn build_wix_app_installer(
       &msi_output_path,
     )?;
     rename(&msi_output_path, &msi_path)?;
-    try_sign(&msi_path, settings)?;
-    output_paths.push(msi_path);
```

Comment:
this line shouldn't be removed

Reply:
@amrbashir its still there, it just comes from sign_params now right below it. I added back `output_paths.push(msi_path);`